### DISCUSSION TITLE
Nuxt

### DISCUSSION THREAD

**Discussion by kylegl** [Colonel-Sandvich/trpc-vue-query#6] [General]
Is this for nuxt or just vue?

**Comment by Colonel-Sandvich** [comment]
I have this working in my own nuxt project right now. I'll get round to making a nuxt starter example this weekend hopefully. I believe SSR doesn't work yet though.

**Comment by Colonel-Sandvich** [comment]
@kylegl I released v0.2.0 not long ago. Refer to `/examples/nuxt-starter` to see what boilerplate is needed to get setup.
Open an issue or discussion if you find any problems!

**Reply by kylegl** [reply]
Cool I'll check it out!
